tool/tctl/common/auth_command.go

```diff
@@ -176,9 +176,9 @@ func (a *AuthCommand) ExportAuthorities(ctx context.Context, client auth.ClientI
 		return a.exportTLSAuthority(ctx, client, types.UserCA, true)
 	}
 
-	// if no --type flag is given, export all types
+	// if no --type flag is given, export HostCA and UserCA.
 	if a.authType == "" {
-		typesToExport = []types.CertAuthType{types.HostCA, types.UserCA, types.DatabaseCA}
```

Comment:
I think that `types.DatabaseCA` should be still present in `typesToExport` list. If we remove it there won't be ability to export it.

Reply:
Here is the handler for `--type=db` https://github.com/gravitational/teleport/pull/13177/files#diff-1da7bc0f8a6f715d9554a2bf8b7a97551d546cb187ab2135b78df3cf7636f21dL173